pkg/container-runtime/installer.go

```diff
@@ -35,8 +50,21 @@ type Info struct {
 	CRISocket string
 }
 
-// 实现
-type dockerInstaller struct{}
+func NewInstaller(conf Config, driver infradriver.InfraDriver) Installer {
+	if conf.Type == Docker {
+		dockerinstall := &DockerInstaller{
+			rootfs: driver.GetClusterRootfs(),
+			driver: driver,
+		}
+		return dockerinstall
+	}
 
-//type containerdInstaller struct{}
-type customContainerRuntimeInstaller struct{}
+	if conf.Type == Containerd {
+		containerdInstaller := &ContainerdInstaller{
+			rootfs: driver.GetClusterRootfs(),
+			driver: driver,
+		}
+		return containerdInstaller
+	}
+	return nil
```

Comment:
this line will cause severe panic. This function cannot return a nil. This function should be defined to return two parameter
```
func NewInstaller(conf Config, driver infradriver.InfraDriver) (Installer, error) {
```
When the input config type is neither docker nor containerd. This function should return an error.